chrysom/README.md

```diff
@@ -0,0 +1,36 @@
+# chrysom
+The client library for communicating with Argus over HTTP.
+[![PkgGoDev](https://pkg.go.dev/github.com/xmidt-org/argus@v0.5.1/chrysom)](https://pkg.go.dev/github.com/xmidt-org/argus@v0.5.1/chrysom)
+
+## Summary
+This package enables CRUD operations with Argus: 
+
+### CRUD Operations
+
+Gets all items that belong to a given owner
+```
+func (c *Client) GetItems(ctx context.Context, owner string) (Items, error)
+```
+Updates an item if the item exists and the ownership matches. If the item does not exist then a new item will be created
+```
+func (c *Client) PushItem(ctx context.Context, owner string, item model.Item) (PushResult, error)
+```
+Removes the item if it exists and returns the data associated to it.
+```
+func (c *Client) RemoveItem(ctx context.Context, id, owner string) (model.Item, error)
+```
+
+### Listener
+The client contains a listener that will listen for item updates from Argus on an interval based on the client configuration. 
+``` PullInterval```
```

Comment:
I'm not sure what this is here for? But it's inverting/breaking the code snippets after this.